src/utils/index.js

```diff
@@ -456,4 +457,23 @@ export function removeNullProperties(object) {
     }
     return acc;
   }, {});
+
+  for (const key in myObject) {
```

Comment:
Thanks for the suggestion.  I took a look at this, and its a bit harder for me to parse through. 
i made this fiddle here https://jsfiddle.net/72rgut53/ to see if it would work, and we'd still need to delete the key from the object.